src/create_disk.sh

```diff
@@ -396,11 +396,15 @@ cat > $rootfs/.coreos-aleph-version.json << EOF
 }
 EOF
 
-install_uefi() {
+install_via_bootupd() {
```

Comment:
in case this causes regressions maybe we should make this a image.yaml opt-in knob and slowly roll it out?

i.e. I think I'd be happy to enable it in `next` now and then let that flow into testing and stable along with the f39 rebase in october.

Reply:
we have a few other image.yaml knobs for the AMI changes that we're already going to shepherd through with the f39 update so we could just manage them together probably.

Reply:
> in case this causes regressions maybe we should make this a image.yaml opt-in knob and slowly roll it out?


Mmmm....not opposed...if you feel slightly strongly.  Wouldn't be too hard.

OTOH, BIOS boot basically either works or it doesn't (and I've tested this now, and I think Huijing did a good bit of testing in development too).

Reply:
interesting in other takes here: cc @travier, @jlebon

Reply:
one other piece of info.. if we merge this as is we'd need to tag a COSA from before this lands and use it to build the next `stable` release because a new enough bootupd won't be in that payload IIUC. for the next `testing` and `next` we can fast-track the bootupd pending update.

Reply:
Actually the more obvious problem is it won't work at all with RHCOS for some time unless we fast track it there...so yeah we need a config knob.